_includes/docs/2.5/ajax-api.html

```diff
@@ -1075,7 +1121,14 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
   "status" : "error"
 }
 </pre>
-
+An example failure response for invalid schedule period:
+<pre class="code">
+{
+  "message" : "PROJECT_NAME.FLOW_NAME scheduled.",
+  "error" : "Invalid schedule period unit 'A",
+  "status" : "success"
```

Comment:
Ah, that's unfortunate. I've opened https://github.com/azkaban/azkaban/issues/407 for this issue.